CMakeLists.txt

```diff
@@ -0,0 +1,125 @@
+project (nice)
```

Comment:
Why do you want to add cmake support? What platform are you building on? The existing autotools build system should work — if there's a particular platform it doesn't work on, then I think I'd rather fix autotools to work on that platform rather than supporting another build system in parallel.